src/content/en/fundamentals/media/_code/tiny-fullscreen-shim.js

```diff
@@ -0,0 +1,21 @@
+HTMLElement.prototype.requestFullscreen = HTMLElement.prototype.requestFullscreen || HTMLElement.prototype.webkitRequestFullscreen;
+HTMLDocument.prototype.exitFullscreen = HTMLDocument.prototype.exitFullscreen || HTMLDocument.prototype.webkitExitFullscreen;
+
+if (!('fullscreenElement' in document)) {
+  Object.defineProperty(document, 'fullscreenElement', {
+    get: function() {
+      return document.mozFullScreenElement || document.msFullscreenElement || document.webkitFullscreenElement;
+    }
+  });
+}
+
+for (var prefixedFullscreenChangeEvent of ['webkitfullscreenchange']) {
+  document.addEventListener(prefixedFullscreenChangeEvent, function(event) {
+    event.stopPropagation();
+    event.stopImmediatePropagation();
+
+    var fullscreenChange = document.createEvent('Event');
+    fullscreenChange.initEvent('fullscreenchange', true /*bubbles */, false /* cancelable */);
+    event.target.dispatchEvent(fullscreenChange);
```

Comment:
The event target of the webkitfullscreenchange event is an element, but for the unprefixed fullscreenchange event it is the document. However, it's not certain that this will remain so, in order to actually get the unprefixed API shipped we may have to change it.

The best bet for now is to listen for all the events on document, since the prefixed events bubble.